packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.test.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Identifiers } from './id-field.component';
+import { Resources, ResourcesContext } from '../../../offline.resources';
+import { Form, Formik } from 'formik';
+import { PatientRegistrationContext } from '../../patient-registration-context';
+import { useConfig } from '@openmrs/esm-framework';
+import { openmrsID } from '../__mocks__/identifiers.mock';
+import { mockedIdentifierTypes } from '../__mocks__/identifier-types.mock';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  ...jest.requireActual('@openmrs/esm-framework'),
+  useConfig: jest.fn(),
+}));
+
+describe('Identifiers', () => {
+  const mockResourcesContextValue = {
+    addressTemplate: [],
+    currentSession: {
+      authenticated: true,
+      sessionId: 'JSESSION',
+      currentProvider: { uuid: 'provider-uuid', identifier: 'PRO-123' },
+    },
+    relationshipTypes: [],
+    identifierTypes: [...mockedIdentifierTypes],
+  } as Resources;
+
+  (useConfig as jest.Mock).mockImplementation(() => ({
```

Comment:
Instead of this, you can mock the implementation within the above jest.mock() function it self. Refer to [this](https://github.com/openmrs/openmrs-esm-patient-chart/blob/5d2baf9f1ff021ff85ad5a9468237f4a794e3d45/packages/esm-patient-programs-app/src/programs/programs-overview.test.tsx#L20)